server/tso/allocator_manager.go

```diff
@@ -1081,6 +1099,35 @@ func (am *AllocatorManager) setGRPCConn(newConn *grpc.ClientConn, addr string) {
 	am.localAllocatorConn.clientConns[addr] = newConn
 }
 
+func (am *AllocatorManager) transferLocalAllocator(dcLocation string, targetServerID uint64) error {
+	serverID := targetServerID
+	nextLeaderKey := am.nextLeaderKey(dcLocation)
+	// Grant a etcd lease with checkStep * 1.5
+	nextLeaderLease := clientv3.NewLease(am.member.Client())
+	ctx, cancel := context.WithTimeout(am.member.Client().Ctx(), etcdutil.DefaultRequestTimeout)
+	leaseResp, err := nextLeaderLease.Grant(ctx, int64(checkStep.Seconds()*1.5))
+	cancel()
+	if err != nil {
+		err = errs.ErrEtcdGrantLease.Wrap(err).GenWithStackByCause()
+		log.Error("failed to grant the lease of the next leader id key", errs.ZapError(err))
```

Comment:
Maybe we should specify `the next leader id` as `the next dc-1 tso allocator leader id`. Otherwise, it may confuse the reader to misunderstand the leader as a PD leader while reading the log.